tests/baselines/reference/user/lodash.log

```diff
@@ -41,6 +41,9 @@ node_modules/lodash/_baseFlatten.js(19,29): error TS2322: Type '(value: any) =>
 node_modules/lodash/_baseFlatten.js(24,22): error TS2349: Cannot invoke an expression whose type lacks a call signature. Type 'Boolean' has no compatible call signatures.
 node_modules/lodash/_baseFlatten.js(24,22): error TS2532: Object is possibly 'undefined'.
 node_modules/lodash/_baseFlatten.js(24,22): error TS2722: Cannot invoke an object which is possibly 'undefined'.
+node_modules/lodash/_baseHas.js(16,56): error TS2345: Argument of type 'string | any[]' is not assignable to parameter of type 'string | number | symbol'.
+  Type 'any[]' is not assignable to type 'string | number | symbol'.
+    Type 'any[]' is not assignable to type 'symbol'.
```

Comment:
`hasOwnProperty` definitely *doesn't* work with arrays. Good catch, strict `call`!

Reply:
I guess it would be nice if it, like, lifted itself when passed an array, but Javascript sure isn't Perly enough for that.